writeengine/wrapper/writeengine.cpp

```diff
@@ -760,6 +933,8 @@ int WriteEngineWrapper::deleteRow(const TxnID& txnid, const vector<CSCTypesList>
     string           tmpStr("");
     vector<DctnryStructList> dctnryExtentsStruct;
 
+    m_opType = DELETE;
```

Comment:
How this assignment differs from the same from line 988?

Reply:
I deleted this one because it and assignment below are duplicates.